addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.js

```diff
@@ -3,29 +3,28 @@
 import { registerMessagingComponent } from '@mail/utils/messaging_component';
 
 const { Component } = owl;
-const { useRef } = owl.hooks;
+const { onMounted, useRef } = owl.hooks;
 
 export class ActivityMarkDonePopover extends Component {
 
     /**
      * @override
      */
-    constructor(...args) {
-        super(...args);
+    setup() {
+        super.setup();
         this._feedbackTextareaRef = useRef('feedbackTextarea');
+        onMounted(() => {
+            this._feedbackTextareaRef.el.focus();
+            if (this.activity.feedbackBackup) {
+                this._feedbackTextareaRef.el.value = this.activity.feedbackBackup;
+            }
+        });
```

Comment:
limit diff by keeping method in-place (+ rename to not conflict with lifecycle methods)